lib/results.js

```diff
@@ -157,8 +158,12 @@ function encodeResult (res, count) {
     if (res.at) {
         output += inner + 'at: ' + res.at + '\n';
     }
-    if (res.operator === 'error' && res.actual && res.actual.stack) {
-        var lines = String(res.actual.stack).split('\n');
+
+    var actualStack = res.actual && res.actual.stack;
+    var errorStack = res.error && res.error.stack;
+    var stack = defined(actualStack, errorStack);
```

Comment:
why `defined` and not `actualStack || errorStack`? Are we worried about a stack trace that's falsy but not `undefined`?